agent/exec/container/executor.go

```diff
@@ -22,10 +22,10 @@ func NewExecutor(client engineapi.APIClient) exec.Executor {
 }
 
 // Describe returns the underlying node description from the docker client.
-func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, error) {
+func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, *api.GossipStatus, error) {
```

Comment:
What? Why are we just adding this wherever?

`Describe` returns a `NodeDescription`.